_gsocorgs/2020/slac.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
SLAC organization is missing the _description:_ field

Reply:
:+1: Done